<!DOCTYPE html>
<html>
<head>
    <title>FISCHER SK WEB Suite Report</title>
    <meta name="generator" content="HTMLTestRunner 1.0.17"/>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    <link rel="stylesheet" href="stylesheet.css">
    <script src="script.js"></script>
    <style>
        
    </style>
    <script>
        
    </script>
</head>
<body>
<div class='heading'>
    <h1 style="margin-left:1ex;">FISCHER SK WEB Suite Report</h1>
    
    <p class='attribute'><strong>Start Time : </strong> 2023-08-15 15:14:30</p>
    
    <p class='attribute'><strong>Duration : </strong> 0:11:49.621011</p>
    
    <p class='attribute'><strong>Status : </strong> Pass 2 Error 24</p>
    
    <p class='attribute'><strong>Descrition : </strong> FISCHER SK WEB Suite Report</p>
    
    <p class='attribute'><strong>Tested By : </strong> Unknown</p>
    
</div>
<div>
    <p id='show_detail_line'><b>Show</b>
        <button><a href='javascript:showCase(0)' style="color:#660066;">Summary</a></button>
        <button><a href='javascript:showCase(1)' style="color:#660066;">Failed</a></button>
        <button><a href='javascript:showCase(2)' style="color:#660066;">Passed</a></button>
        <button><a href='javascript:showCase(3)' style="color:#660066;">All</a></button>
    </p>
    <table id='result_table'>
        <colgroup>
            <col align='left'/>
            <col align='right'/>
            <col align='right'/>
            <col align='right'/>
            <col align='right'/>
            <col align='right'/>
        </colgroup>
        <tr id='header_row'>
            <td>Test Group/Test case</td>
            <td>Count</td>
            <td>Pass</td>
            <td>Fail</td>
            <td>Error</td>
            <td>View</td>
        </tr>
        
        <tr class='errorClass' style="font-weight:bold;color:white;">
            <td><a href="javascript:showClassDetail('c1','1')">FWSK_Automation_Local_Deploy_PyCharm.CovidInfo_D.TestCovidInfo_D</a></td>
            <td>1</td>
            <td>0</td>
            <td>0</td>
            <td>1</td>
            <td>
                <button><a href="javascript:showClassDetail('c1','1')"
                           style="color:#660066;">Detail</a></button>
            </td>
        </tr>
            
            <tr id="ft1.1" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft1.1')">test_covidInfo_D</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft1.1')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft1.1' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft1.1').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\CovidInfo_D.py", line 10, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 63, in __extract_zip
    archive = zip_class(archive_file.file_path)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1269, in __init__
    self._RealGetContents()
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1336, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
        
        <tr class='passClass' style="font-weight:bold;color:white;">
            <td><a href="javascript:showClassDetail('c2','1')">FWSK_Automation_Local_Deploy_PyCharm.Detail_D.TestDetailHotelu_D</a></td>
            <td>1</td>
            <td>1</td>
            <td>0</td>
            <td>0</td>
            <td>
                <button><a href="javascript:showClassDetail('c2','1')"
                           style="color:#660066;">Detail</a></button>
            </td>
        </tr>
            
            <tr id="pt2.1" class='hiddenRow'>
                <td class='passCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_pt2.1')">test_detail_D</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_pt2.1')" style="color:#660066;">
                                PASS
                            </a></button>
                        <div id='div_pt2.1' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_pt2.1').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre>2023-08-15 15:14:58.798838 : <selenium.webdriver.remote.webelement.WebElement (session="39821dc624f5db1568669cb233f25041", element="6BB14E67210C3ACE3859E97D4F87B95C_element_117")>
2023-08-15 15:14:59.011975 : https://fischersk.web1.dtweb.cz//egypt/egypt-hurghada/hurghada/sentido-mamlouk-palace?AC1=2&D=653|819|724&DD=2023-10-01&DP=1837&DPR=FISCHER+SK+ATCOM&DS=256&GIATA=78630&HID=145866&IC1=0&IFM=0&ILM=0&KC1=0&MNN=7&MT=5&NN=7&PID=HRG22103&RD=2023-10-08&TO=1837&acm1=2&df=2023-10-01|2023-10-15&nnm=7&tt=1&ttm=1#/preh%C4%BEad
</pre>
                            <pre style="color: red;"></pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
        
        <tr class='errorClass' style="font-weight:bold;color:white;">
            <td><a href="javascript:showClassDetail('c3','4')">FWSK_Automation_Local_Deploy_PyCharm.Detail_C.TestDetailHotelu_C</a></td>
            <td>4</td>
            <td>1</td>
            <td>0</td>
            <td>3</td>
            <td>
                <button><a href="javascript:showClassDetail('c3','4')"
                           style="color:#660066;">Detail</a></button>
            </td>
        </tr>
            
            <tr id="pt3.1" class='hiddenRow'>
                <td class='passCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_pt3.1')">test_detail_fotka</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_pt3.1')" style="color:#660066;">
                                PASS
                            </a></button>
                        <div id='div_pt3.1' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_pt3.1').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre>2023-08-15 15:15:09.887065 : <selenium.webdriver.remote.webelement.WebElement (session="a518dde0341b74b4134ed0fcc906ea46", element="4B5F22B4AED6D704E0E4E32859C496BF_element_142")>
2023-08-15 15:15:20.204032 : its ok
2023-08-15 15:15:20.209018 : https://img.fischer.cz/hotels/egypt/oblast-hurghada/hurghada/sentido-mamlouk-palace-resort-a-spa/18033016.jpg
</pre>
                            <pre style="color: red;"></pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
            <tr id="ft3.2" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft3.2')">test_detail_terminy_filtr_meal</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft3.2')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft3.2' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft3.2').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\Detail_C.py", line 86, in test_detail_terminy_filtr_meal
    self.driver.get(URL_detail)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\selenium\webdriver\remote\webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 299.896
  (Session info: chrome=115.0.5790.170)

</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
            <tr id="ft3.3" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft3.3')">test_detail_terminy_filtr_meal</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft3.3')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft3.3' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft3.3').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\Detail_C.py", line 39, in tearDown
    tearDown(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 73, in tearDown
    print(self.driver.current_url)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\selenium\webdriver\remote\webdriver.py", line 669, in current_url
    return self.execute(Command.GET_CURRENT_URL)['value']
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 300.000
  (Session info: chrome=115.0.5790.170)

</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
            <tr id="ft3.4" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft3.4')">test_detail_terminy_filtr_airport</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft3.4')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft3.4' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft3.4').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\Detail_C.py", line 36, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
        
        <tr class='errorClass' style="font-weight:bold;color:white;">
            <td><a href="javascript:showClassDetail('c4','1')">FWSK_Automation_Local_Deploy_PyCharm.DetskeKluby_D.TestDetskeKluby_D</a></td>
            <td>1</td>
            <td>0</td>
            <td>0</td>
            <td>1</td>
            <td>
                <button><a href="javascript:showClassDetail('c4','1')"
                           style="color:#660066;">Detail</a></button>
            </td>
        </tr>
            
            <tr id="ft4.1" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft4.1')">test_kluby_D</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft4.1')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft4.1' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft4.1').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\DetskeKluby_D.py", line 10, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
        
        <tr class='errorClass' style="font-weight:bold;color:white;">
            <td><a href="javascript:showClassDetail('c5','1')">FWSK_Automation_Local_Deploy_PyCharm.dovolena_D.TestDovolena_D</a></td>
            <td>1</td>
            <td>0</td>
            <td>0</td>
            <td>1</td>
            <td>
                <button><a href="javascript:showClassDetail('c5','1')"
                           style="color:#660066;">Detail</a></button>
            </td>
        </tr>
            
            <tr id="ft5.1" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft5.1')">test_dovolena_D</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft5.1')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft5.1' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft5.1').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\dovolena_D.py", line 13, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
        
        <tr class='errorClass' style="font-weight:bold;color:white;">
            <td><a href="javascript:showClassDetail('c6','1')">FWSK_Automation_Local_Deploy_PyCharm.FM_D.TestFMexotika_D</a></td>
            <td>1</td>
            <td>0</td>
            <td>0</td>
            <td>1</td>
            <td>
                <button><a href="javascript:showClassDetail('c6','1')"
                           style="color:#660066;">Detail</a></button>
            </td>
        </tr>
            
            <tr id="ft6.1" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft6.1')">test_FM_exotika_D</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft6.1')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft6.1' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft6.1').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\FM_D.py", line 49, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
        
        <tr class='errorClass' style="font-weight:bold;color:white;">
            <td><a href="javascript:showClassDetail('c7','2')">FWSK_Automation_Local_Deploy_PyCharm.fulltext_C.Test_Fulltext_C</a></td>
            <td>2</td>
            <td>0</td>
            <td>0</td>
            <td>2</td>
            <td>
                <button><a href="javascript:showClassDetail('c7','2')"
                           style="color:#660066;">Detail</a></button>
            </td>
        </tr>
            
            <tr id="ft7.1" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft7.1')">test_fulltext_naseptavac</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft7.1')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft7.1' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft7.1').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\fulltext_C.py", line 21, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
            <tr id="ft7.2" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft7.2')">test_fulltext_results_status_check</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft7.2')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft7.2' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft7.2').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\fulltext_C.py", line 21, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
        
        <tr class='errorClass' style="font-weight:bold;color:white;">
            <td><a href="javascript:showClassDetail('c8','1')">FWSK_Automation_Local_Deploy_PyCharm.groupsearch_D.Test_Groupsearch_D</a></td>
            <td>1</td>
            <td>0</td>
            <td>0</td>
            <td>1</td>
            <td>
                <button><a href="javascript:showClassDetail('c8','1')"
                           style="color:#660066;">Detail</a></button>
            </td>
        </tr>
            
            <tr id="ft8.1" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft8.1')">test_groupsearch_D</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft8.1')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft8.1' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft8.1').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\groupsearch_D.py", line 64, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
        
        <tr class='errorClass' style="font-weight:bold;color:white;">
            <td><a href="javascript:showClassDetail('c9','1')">FWSK_Automation_Local_Deploy_PyCharm.HP_D.TestHP_D</a></td>
            <td>1</td>
            <td>0</td>
            <td>0</td>
            <td>1</td>
            <td>
                <button><a href="javascript:showClassDetail('c9','1')"
                           style="color:#660066;">Detail</a></button>
            </td>
        </tr>
            
            <tr id="ft9.1" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft9.1')">test_homePage_D</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft9.1')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft9.1' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft9.1').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\HP_D.py", line 11, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
        
        <tr class='errorClass' style="font-weight:bold;color:white;">
            <td><a href="javascript:showClassDetail('c10','1')">FWSK_Automation_Local_Deploy_PyCharm.LM_D.TestLM_D</a></td>
            <td>1</td>
            <td>0</td>
            <td>0</td>
            <td>1</td>
            <td>
                <button><a href="javascript:showClassDetail('c10','1')"
                           style="color:#660066;">Detail</a></button>
            </td>
        </tr>
            
            <tr id="ft10.1" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft10.1')">test_lM_isDisplayed</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft10.1')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft10.1' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft10.1').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\LM_D.py", line 12, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
        
        <tr class='errorClass' style="font-weight:bold;color:white;">
            <td><a href="javascript:showClassDetail('c11','1')">FWSK_Automation_Local_Deploy_PyCharm.pobocky.TestPobocky_D</a></td>
            <td>1</td>
            <td>0</td>
            <td>0</td>
            <td>1</td>
            <td>
                <button><a href="javascript:showClassDetail('c11','1')"
                           style="color:#660066;">Detail</a></button>
            </td>
        </tr>
            
            <tr id="ft11.1" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft11.1')">test_pobocky_D</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft11.1')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft11.1' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft11.1').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\pobocky.py", line 8, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
        
        <tr class='errorClass' style="font-weight:bold;color:white;">
            <td><a href="javascript:showClassDetail('c12','1')">FWSK_Automation_Local_Deploy_PyCharm.SDO_D.TestSDO_D</a></td>
            <td>1</td>
            <td>0</td>
            <td>0</td>
            <td>1</td>
            <td>
                <button><a href="javascript:showClassDetail('c12','1')"
                           style="color:#660066;">Detail</a></button>
            </td>
        </tr>
            
            <tr id="ft12.1" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft12.1')">test_SDO_D</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft12.1')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft12.1' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft12.1').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\SDO_D.py", line 8, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
        
        <tr class='errorClass' style="font-weight:bold;color:white;">
            <td><a href="javascript:showClassDetail('c13','5')">FWSK_Automation_Local_Deploy_PyCharm.SRL_C.Test_SRL_C</a></td>
            <td>5</td>
            <td>0</td>
            <td>0</td>
            <td>5</td>
            <td>
                <button><a href="javascript:showClassDetail('c13','5')"
                           style="color:#660066;">Detail</a></button>
            </td>
        </tr>
            
            <tr id="ft13.1" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft13.1')">test_SRL_sort_cheapest</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft13.1')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft13.1' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft13.1').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\SRL_C.py", line 16, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
            <tr id="ft13.2" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft13.2')">test_SRL_sort_expensive</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft13.2')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft13.2' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft13.2').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\SRL_C.py", line 16, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
            <tr id="ft13.3" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft13.3')">test_SRL_map</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft13.3')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft13.3' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft13.3').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\SRL_C.py", line 16, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
            <tr id="ft13.4" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft13.4')">test_SRL_filtr_strava</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft13.4')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft13.4' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft13.4').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\SRL_C.py", line 16, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
            <tr id="ft13.5" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft13.5')">test_srl_C</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft13.5')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft13.5' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft13.5').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\SRL_C.py", line 16, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
        
        <tr class='errorClass' style="font-weight:bold;color:white;">
            <td><a href="javascript:showClassDetail('c14','1')">FWSK_Automation_Local_Deploy_PyCharm.SRL_D.TestSRL_D</a></td>
            <td>1</td>
            <td>0</td>
            <td>0</td>
            <td>1</td>
            <td>
                <button><a href="javascript:showClassDetail('c14','1')"
                           style="color:#660066;">Detail</a></button>
            </td>
        </tr>
            
            <tr id="ft14.1" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft14.1')">test_SRL_D</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft14.1')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft14.1' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft14.1').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\SRL_D.py", line 107, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
        
        <tr class='errorClass' style="font-weight:bold;color:white;">
            <td><a href="javascript:showClassDetail('c15','3')">FWSK_Automation_Local_Deploy_PyCharm.HP_C.Test_HP_C</a></td>
            <td>3</td>
            <td>0</td>
            <td>0</td>
            <td>3</td>
            <td>
                <button><a href="javascript:showClassDetail('c15','3')"
                           style="color:#660066;">Detail</a></button>
            </td>
        </tr>
            
            <tr id="ft15.1" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft15.1')">test_HP_zlutak_to_SRL</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft15.1')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft15.1' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft15.1').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\HP_C.py", line 50, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
            <tr id="ft15.2" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft15.2')">test_HP_zlutak_to_groupsearch</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft15.2')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft15.2' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft15.2').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\HP_C.py", line 50, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
            <tr id="ft15.3" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft15.3')">test_HP_bannery_check</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft15.3')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft15.3' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft15.3').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\HP_C.py", line 50, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
        
        <tr class='errorClass' style="font-weight:bold;color:white;">
            <td><a href="javascript:showClassDetail('c16','1')">FWSK_Automation_Local_Deploy_PyCharm.SRL_results_comparer.Test_SRL_C_comparer</a></td>
            <td>1</td>
            <td>0</td>
            <td>0</td>
            <td>1</td>
            <td>
                <button><a href="javascript:showClassDetail('c16','1')"
                           style="color:#660066;">Detail</a></button>
            </td>
        </tr>
            
            <tr id="ft16.1" class='none'>
                <td class='errorCase'>
                    <div class='testcase'>
                        <a class="popup_link" onfocus='this.blur();'
                           href="javascript:showTestDetail('div_ft16.1')">test_SRL_number_of_results_comparer</a>
                    </div>
                </td>
                <td colspan='5' align='center'>
                    <div>
                        <!--css div popup start-->
                        <button>
                            <a class="popup_link" onfocus='this.blur();'
                               href="javascript:showTestDetail('div_ft16.1')" style="color:#660066;">
                                ERROR
                            </a></button>
                        <div id='div_ft16.1' class="popup_window">
                            <div style='text-align: right; color:red;cursor:pointer'>
                                <a onfocus='this.blur();'
                                   onclick="document.getElementById('div_ft16.1').style.display = 'none'; "
                                   style="color:red;">
                                    [x]</a>
                            </div>
                            <pre></pre>
                            <pre style="color: red;">Traceback (most recent call last):
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 65, in __extract_zip
    archive.extractall(to_directory)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1647, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\KADOUN\AppData\Local\Programs\Python\Python38\lib\zipfile.py", line 1701, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\SRL_results_comparer.py", line 38, in setUp
    setUp(self)
  File "C:\Users\KADOUN\Desktop\Automatization_Project_Master_QA\FWSK_Automation_Local_Deploy_PyCharm\to_import.py", line 34, in setUp
    self.driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\manager.py", line 41, in _get_driver_binary_path
    binary_path = self._cache_manager.save_file_to_cache(driver, file)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 54, in save_file_to_cache
    files = self.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\driver_cache.py", line 49, in unpack_archive
    return self._file_manager.unpack_archive(archive, path)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 57, in unpack_archive
    return self.__extract_zip(archive_file, target_dir)
  File "C:\Users\KADOUN\AppData\Roaming\Python\Python38\site-packages\webdriver_manager\core\file_manager.py", line 81, in __extract_zip
    os.replace(source, destination)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\KADOUN\\.wdm\\drivers\\chromedriver\\win64\\115.0.5790.170\\chromedriver.exe'
</pre>
                        </div>
                        <!--css div popup end-->
                    </div>
                </td>
            </tr>
            
        

        <tr id='total_row'>
            <td>Total</td>
            <td>26</td>
            <td>2</td>
            <td>0</td>
            <td>24</td>
            <td>&nbsp;</td>
        </tr>
    </table>
</div>
<br><br>
<div id='ending'><b>Time Elapsed: </b>0:11:49.621011</div>
</body>
</html>